Misc/NEWS.d/next/Library/2019-05-26-01-20-06.bpo-37047.K9epi8.rst

```diff
@@ -0,0 +1,2 @@
+Handle late binding and attribute access in AsyncMock setup for
+autospeccing. Document newly implemented async methods in MagicMock.
```

Comment:
If you want to make this a link you can convert the AsyncMock and MagicMock to
:class:`unittest.mock.AsyncMock` and :class:`unittest.mock.MagicMock`

Reply:
Done. Thanks.